llvm/test/tools/llvm-readobj/ELF/relr-relocs.test

```diff
@@ -1,6 +1,7 @@
 ## This is a test to test how SHT_RELR sections are dumped.
 
 # RUN: yaml2obj --docnum=1 %s -o %t1
+# RUN: llvm-objcopy --strip-all %t1 %t1.stripped
```

Comment:
I think it would be easier to follow if this line were to be moved down to where `%t1.stripped` is actually used.

Reply:
Moved